.travis.yml

```diff
@@ -83,11 +77,11 @@ before_install:
       echo extension = ldap.so >> $INI
       echo extension = redis.so >> $INI
       echo extension = memcached.so >> $INI
-      [[ $PHP = 5.* ]] && echo extension = mongo.so >> $INI
-      [[ $PHP = 5.* ]] && echo extension = memcache.so >> $INI
+      #echo extension = mongodb.so >> $INI
```

Comment:
the test suite does not pass yet with mongodb - but this is not master specific so should be handled on a lower branch